tests/integration/operator/testdata/Dockerfile.customAgent

```diff
@@ -0,0 +1,6 @@
+ARG PORTER_VERSION
+ARG REGISTRY
+FROM $REGISTRY/porter-agent:$PORTER_VERSION
+
+COPY --chown=65532:65532 bin/plugins/kubernetes/*/kubernetes-linux-amd64 /app/.porter/plugins/kubernetes/kubernetes
```

Comment:
This was because we were always cleaning up our bin before building this so there was only 1 version under /bin/plugins/kubernetes/ but yeah if there's multiple versions under there then this would break. I'll try to clean that up a little bit, not sure if having multiple versions under bin/plugins/kubernetes is a valid use case. Some of this stuff is trying to keep current build process in place and add on top of it without rewriting the whole thing